interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a version control software to manage code modifications. Github is a web host for code that is managed via git commands.
 
-Researched answer:
+Researched answer: No significant differences between my and the researched answer.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: !!, Array.isArray, ==, ===, &&, ||, !=, !==, 
```

Comment:
If you take a look at JavaScript Introduction section in the syllabus it covers this. Logical operators return boolean values. To include loose equality and strict equality (: